python/tweets/tweetloc.py

```diff
@@ -0,0 +1,70 @@
+# This is a "stream listener" - it catches live tweets from the Twitter firehose
+# (the "firehose" is a small proportion of all tweets which is accessible via the
+# Twitter API).
+
+
+# Import modules (you may need to `pip install` these at the command line):
+# - tweepy, for interacting with the Twitter API
+# - json, for working with the tweets themselves (they are returned as json objects)
+# - pymongo, for connecting to a mongoDB mLab database where we'll store the tweets
+import tweepy, json, pymongo
+
+
+# Set up API - replace keys with your own: https://apps.twitter.com/
+CONSUMER_KEY = "<CONSUMER_KEY>"
+CONSUMER_SECRET = "<CONSUMER_SECRET>"
+ACCESS_KEY = "<ACCESS_KEY>"
+ACCESS_SECRET = "<ACCESS_TOKEN>"
+auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+auth.set_access_token(ACCESS_KEY, ACCESS_SECRET)
+api = tweepy.API(auth)
+
+
+# Define a class called LocationListener which extends tweepy's StreamListener class
+class LocationListener(tweepy.StreamListener):
+
+    def __init__(self, api):
+    
+        # Use the API we have set up
+        self.api = api
+        
+        # We use super() to also run the __init__() of the parent class, i.e StreamListener
+        super(tweepy.StreamListener, self).__init__()
```

Comment:
I think calls to super tend to be the first line in a constructor

Reply:
Should be. I'll move this. Remembered to add it quite late and didn't really take care about where I put it!